src/ToDo.UI/ToDo.UI.shproj

```diff
@@ -10,5 +10,7 @@
   <PropertyGroup />
   <Import Project="ToDo.UI.projitems" Label="Shared" />
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.CSharp.targets" />
-
+  <ItemGroup>
+    <_Globbed_Compile Remove="Controls\NavigationViewRequestHandler.cs" />
+  </ItemGroup>
```

Comment:
This can be safely removed from commit/PR.

Reply:
removed